flags.go

```diff
@@ -81,10 +75,36 @@ var (
 		Value:  3,
 		EnvVar: envVarPrefix + "RETRY_SYNC_N_TIMES",
 	}
+	EcdsaPrivateKeyFlag = cli.StringFlag{
+		Name:   "ecdsa-private-key",
+		Usage:  "Ethereum ecdsa private key",
+		EnvVar: envVarPrefix + "ECDSA_PRIVATE_KEY",
+	}
+	// Fireblocks flags
+	FireblocksAPIKeyFlag = cli.StringFlag{
+		Name:   "fireblocks-api-key",
+		Usage:  "Fireblocks API Key",
+		EnvVar: envVarPrefix + "FIREBLOCKS_API_KEY",
```

Comment:
I'm guessing we have a check that exactly one of these 2 has to be set? Can we add a comment saying this?
Too bad urfave, afaik, doesn't allow more expressive optional flag logic. Python args for eg allow you to add 2 flags that are required but mutually exclusive (so 1 and only 1 is specified)

Reply:
Yeah added some clarification